### PR TITLE
Feature starting spring sec

### PR DESCRIPTION
**Pull request by FeiYGH**

### CODE REVIEW

File: src/main/java/com/talentpath/shamazin/showItemPage/controllers/AuthController.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        //getting to the empty set
+        toAdd.getRoles().add(roleRepo.findByRole(Role.RoleName.ROLE_USER).get());
+        toAdd = userRepo.saveAndFlush(toAdd);
+        //responseEntity is a wrapper, what's inside is important(here Obj contains a string)
+        return ResponseEntity.ok(new MessageResponse("User has registered successfully!!"));
+    }
+
+
+    @GetMapping("test")
+    public String testSec(){
```
Comment (neveland98) [inline src/main/java/com/talentpath/shamazin/showItemPage/controllers/AuthController.java]: make sure to remove testing code before merging into staging

File: src/main/java/com/talentpath/shamazin/showItemPage/controllers/UserDataController.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+@RequestMapping("/api/userdata")
+public class UserDataController {
+    @Autowired
+    UserRepository userRepo;
+
+    @Autowired
+    RoleRepository roleRepo;
+
+    @GetMapping("/test")
+    public String testUserDataController(){
```
Comment (neveland98) [inline src/main/java/com/talentpath/shamazin/showItemPage/controllers/UserDataController.java:25]: Make sure to remove testing code before merging.

**Review by neveland98 — CHANGES_REQUESTED** [review]
See comments

**Review by jaitken — APPROVED** [review]
LGTM with Nathanial's comments

**Review by neveland98 — APPROVED** [review]

**Pull request merged**